lib/thunderbird_common.pm

```diff
@@ -29,7 +29,7 @@ C<$proto> can be C<pop> or C<imap>.
 C<$account> can be C<internal_account_A> or C<internal_account_B> or C<internal_account_C> or C<internal_account_D>.
 =cut
 sub tb_setup_account {
-    my $hostname = get_var('HOSTNAME');
+    my $hostname = get_var('HOSTNAME') // '';
```

Comment:
nitpicking: it would be better with `//=` as it checks if `get_var('HOSTNAME')` is also defined and not only true
either `my $hostname = get_var('HOSTNAME') // undef;` or `my $hostname = get_var('HOSTNAME') //= '';` wdyt?

Reply:
You don't understand the issue `Fixes 'Use of uninitialized value $hostname in string eq at ...'`
The issue is that later in code is `$hostname` used and because it's `undef` it returns the error.

Reply:
what i understand from this is that `my $hostname = get_var('HOSTNAME') // undef;` is not suitable here. that s fine. And i know that it doesnt make big difference. i just think it is better with _assingment-or_ because is equal in writing with `my $hostname = defined get_var('HOSTNAME') ? get_var('HOSTNAME') : '';`

Reply:
feel free to resolve it if you dont want to this. it was just a suggestion.

Reply:
You just complicate things, I just added empty value to the variable if it does not exist.
There are variants of this tests, thus the variable does not need to be always defined.
Maybe it could be done differently and better. This simply just does fix the issue.